nodejs/week2/warmup/index.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+const _cal = require('./calculator');
+
+//Add 2 routes to your index.js:
+//GET /numbers/add?first=<number here>&second=<number here>.
+//In response send sum (first + second).
+//GET /numbers/multiply/<first number here>/<second number here>.
+//in response send multiplication (first * second).
+
+app.get('/numbers/add/', function(req, res) {
+  let rs = 0;
```

Comment:
Please name your variables, according to naming convention https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md
From the name of the variable another developer should understand for what reason this variable exists.